plugins/com.google.cloud.tools.eclipse.appengine.localserver/src/com/google/cloud/tools/eclipse/appengine/localserver/ui/CloudSdkRuntimeWizardFragment.java

```diff
@@ -146,7 +146,7 @@ public void widgetSelected(SelectionEvent event) {
       }
     });
 
-    File location = new CloudSdkProvider(null).getCloudSdkLocation();
+    File location = new CloudSdkProvider().getCloudSdk().getJavaAppEngineSdkPath().toFile();
```

Comment:
Why do we need toFile here? If getJavaAppEngineSdkPath returns a Path we can just declare location to be that instead, I expect.